client/assets/stylesheets/_p2-vars.scss

```diff
@@ -16,6 +16,10 @@
 	--p2-color-button: #f3f3f3;
 	--p2-color-button-dark: #eaeaea;
 
+	--p2-color-background-dark: #00101c;
```

Comment:
Looks like we already have the color defined in this file above as `--p2-color-text`.

Reply:
Yup! I thought I'd keep it separate though, as we may be tweaking the background color separate from the text color.